.github/CONTRIBUTING.md

```diff
@@ -1,8 +1,20 @@
-This repository is not accepting contributions. We’re releasing the code for others
-to refer to and learn from, but we are not open to pull requests or issues at
-this time.
-
-Khan Academy is a non-profit organization with a mission to provide a free,
-world-class education to anyone, anywhere. You can help us in that mission by
-[donating](https://khanacademy.org/donate) or looking at
-[career opportunities](https://khanacademy.org/careers).
+# Contributing to Terraform Modules
+
+## Development Workflow
+
+1. Fork and clone the repository
+2. Create a feature branch from `main`
+3. Make your changes to modules in `terraform/modules/`
+4. Submit a pull request
+
+## Releases
+
+Releases are created manually using the **Manual Release** GitHub Action:
+
+1. Go to Actions → Manual Release
+2. Select the module name (e.g., `scheduled-function`)
+3. Enter version number (e.g., `1.0.0`)
+4. Add optional release notes
```

Comment:
You should document how to do this: it's by checking in a file with a particular name, right?  Should this file have a particular format?